services/brig/test/integration/API/Team.hs

```diff
@@ -757,25 +760,25 @@ testConnectionSameTeam brig = do
 testCreateUserInternalSSO :: Brig -> Galley -> Http ()
 testCreateUserInternalSSO brig galley = do
   teamid <- snd <$> createUserWithTeam brig
-  let ssoid = UserSSOId "nil" "nil"
+  authId <- liftIO $ AuthSAML <$> generate arbitrary
   -- creating users requires both sso_id and team_id
-  postUser' True False "dummy" True False (Just ssoid) Nothing brig
-    !!! const 400 === statusCode
-  postUser' True False "dummy" True False Nothing (Just teamid) brig
-    !!! const 400 === statusCode
+  -- postUser' True False "dummy" True False (Just ssoid) Nothing brig
+  --   !!! const 400 === statusCode
+  -- postUser' True False "dummy" True False Nothing (Just teamid) brig
+  --   !!! const 400 === statusCode
```

Comment:
This should have been deleted,

| Use `postUser' True False\` instead of 'postUser' if you want to send broken bodies to test error

I decided to refactor `postUser'` to send a proper `NewUser` type instead of a `Aeson.Value`, because I think there is not need to test this in a integration test.

This made these tests obsolete. I'll remove them.